i18n/en.json

```diff
@@ -2,18 +2,18 @@
 	"@metadata": {
 		"authors": [
 			"Southparkfan",
-			"Universal Omega"
+			"Universal Omega",
+			"Alex"
 		]
 	},
 	"action-createwiki": "create wikis",
 	"action-requestwiki": "request wikis",
 	"createwiki": "Create a wiki",
-	"createwiki-defaultmainpage": "__NOTOC__\n== Welcome to {{SITENAME}}! ==\nThis Main Page was created automatically and it seems it hasn't been replaced yet.\n\n=== For the bureaucrat(s) of this wiki ===\nHello, and welcome to your new wiki! Thank you for choosing Miraheze for the hosting of your wiki, we hope you will enjoy our hosting.\n\nYou can immediately start working on your wiki or whenever you want.\n\nNeed help? No problem! We will help you with your wiki as needed. To start, try checking out these helpful links:\n* [[mw:Special:MyLanguage/Help:Contents|MediaWiki guide]] (e.g. navigation, editing, deleting pages, blocking users)\n* [[meta:Special:MyLanguage/FAQ|Miraheze FAQ]]\n* [[meta:Special:MyLanguage/Request features|Request settings changes on your wiki]]. (Extensions, Skin and Logo/Favicon changes should be done through [[Special:ManageWiki]] on your wiki, see [[meta:Special:MyLanguage/ManageWiki|ManageWiki]] for more information.)\n\n==== I still don't understand X! ====\nWell, that's no problem. Even if something isn't explained in the documentation/FAQ, we are still happy to help you. You can find us here:\n* [[meta:Special:MyLanguage/Help center|On our own Miraheze wiki]]\n* On [[phab:|Phabricator]]\n* On [https://miraheze.org/discord Discord]\n* On IRC in #miraheze on irc.libera.chat ([irc://irc.libera.chat/%23miraheze direct link]; [https://web.libera.chat/?channel=#miraheze webchat])\n\n=== For visitors of this wiki ===\nHello, the default Main Page of this wiki (this page) has not yet been replaced by the bureaucrat(s) of this wiki. The bureaucrat(s) might still be working on a Main Page, so please check again later!",
```

Comment:
This can stay here for now. Changing if can be a follow-up due to it being a lot larger.

Reply:
Alright.